Editor/Window/ConnectToFleetInput.cs

```diff
@@ -2,17 +2,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Amazon.GameLift;
 using Amazon.GameLift.Model;
-using AmazonGameLift.Editor;
 using Editor.CoreAPI;
+using Editor.Window;
+using UnityEngine;
 using UnityEngine.UIElements;
 
-namespace Editor.Window
+namespace AmazonGameLift.Editor
 {
     public class ConnectToFleetInput : StatefulInput
     {
         private static IReadOnlyCollection<VisualElement> _fleetVisualElements;
 
+        public string FleetId;
+
+        private static readonly List<string> s_fleetNameList = new();
+
```

Comment:
```suggestion
```
Remove, neither of these appear to be used

Reply:
removed